lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -175,11 +139,9 @@ describe('NxRadio', function() {
       } as Props['inputAttributes']
     });
 
-    expect(component.find('input')).toHaveProp('id', 'garfield');
-    expect(component.find('input')).toHaveProp('name', 'garfield');
-    expect(component.find('input')).toHaveProp('disabled', true);
-    expect(component.find('input')).toHaveClassName('input-classname');
-    expect(component.find('input')).not.toHaveClassName('label-classname');
-    expect(component.find('input')).toHaveProp('checked', true);
+    expect(component.queryByRole('radio')).toHaveAttribute('id', 'garfield');
+    expect(component.queryByRole('radio')).toHaveAttribute('name', 'garfield');
+    expect(component.queryByRole('radio')).toBeDisabled();
+    expect(component.queryByRole('radio')).toBeChecked();
```

Comment:
Custom classnames here were not implementation details.  We want to make sure that `inputAttributes.className` works and that it doesn't get mixed up with the top-level `className`.

Reply:
> Custom classnames here were not implementation details. We want to make sure that `inputAttributes.className` works and that it doesn't get mixed up with the top-level `className`.

Added them back: 395719f19c7504346ded37566cbc6ed07d72f0f2

The test for the class for `isChecked`, `'tm-checked'`, would be the same thing?

Reply:
> The test for the class for isChecked, 'tm-checked', would be the same thing?

The tm classes are a bit of a special case.  at least one of the frontend user analytics tools that Sonatype products use or have used can't pick up the actual checked state of the checkboxes, but can pick up classnames.  So as a workaround we have our checkboxes and radios expose their checked state as a classname.  Since that is an externally-relevant feature, yes it should still be tested